dallinger/experiment_server/utils.py

```diff
@@ -216,8 +216,8 @@ def error_page(
         error_text = "There has been an error and so you are unable to continue, sorry!"
 
     if participant is not None:
-        hit_id = (participant.hit_id,)
```

Comment:
Any idea why this was a tuple?

Reply:
I found it. Combination of erroneous trailing commas + black. 😬 

Pre-blackening:
https://github.com/Dallinger/Dallinger/blame/ff3db33654ee8b33fd525d68530bc34b48e62da6/dallinger/experiment_server/experiment_server.py

Post-blackening:
https://github.com/Dallinger/Dallinger/blame/922f1d9241c18d11ec861750e7356cbe6096a66e/dallinger/experiment_server/experiment_server.py

Reply:
Wow that's an old bug! Looks like a case where black made a subtle error much less subtle, so if the pre-commit were in place when the trailing comma was originally added it would have been caught. That's encouraging at least.